### ISSUE TITLE
define column label in prisma for non-english app

### ISSUE THREAD

**Issue by xiangnanscu** [prisma/prisma#16129]
## Problem

When I define a database column name (in English) in prisma, usually there's a label (in Chinese) for it. 

## Suggested solution

I need prisma to record this info and then can access from prisma client api. I'm wondering if there's something like @label syntax in prisma.

**Comment by SevInf** [comment]
Hi @xiangnanscu!
If you want to use chinese name in your DBMS, but english  name in your source code, you can use [`@@map`](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#map-1) in you schema? Otherwise, could you specify what would be SQL equivalent of this feature?

**Comment by SevInf** [comment]
Found the discussion: https://github.com/prisma/prisma/discussions/15501.

**Comment by xiangnanscu** [comment]
yeah, it's like Django Model Field [verbose_name](https://docs.djangoproject.com/en/4.1/ref/models/fields/#verbose-name), mainly for UI display (form label, table column etc.)

**Comment by xiangnanscu** [comment]
then api like `prisma.user.labelsDict` will be `{"id":"id", "name":"姓名", "permission":"权限"}`. Thus it help UI a lot

**Comment by xiangnanscu** [comment]
@SevInf Hi, I realized the ultimate way to handle this is to allow user pass a js object literal as meta data to column defination. Because other orms in other languages may involve variours options. for example, `verbose_name` and `help_text` in Django. so here is my suggestions:
```prisma
model User {
  name    String? @meta({label: "姓名", help_text:"请输入姓名", min_length:2})
}
```
or
```prisma
model User {
  name    String? @meta.label("姓名") @meta.help_text("请输入姓名") @meta.min_length(2)
}
```
then the client api `prisma.user.meta` will be `{name: {label: "姓名", help_text:"请输入姓名", min_length:2}}`. Then with the help of generators like `prisma-json-schema-generator`, other orms could create model from json made by prisma. This will greatly help prisma embeded into other orms!
